quantum/process_keycode/process_key_override.c

```diff
@@ -190,11 +184,17 @@ const key_override_t *clear_active_override(const bool allow_reregister) {
             del_key(mod_free_replacement);
         } else {
             key_override_printf("NOT KEY 1\n");
-            send_keyboard_report();
             unregister_code(mod_free_replacement);
         }
+        send_keyboard_report();
     }
 
+    // Clear the suppressed mods
+    clear_suppressed_override_mods();
+
+    // Unregister the replacement. First remove the weak override mods
+    clear_weak_override_mods();
```

Comment:
This comment is no longer correct after moving the code.

Reply:
Thank you for your review. I've fixed the comment.